endoreg_db/services/video_import.py

```
@@ -29,51 +52,105 @@ def __init__(self, project_root: Path = None):
+                    except FileNotFoundError:
+                        # Race: lock removed between exists and stat; retry acquire in next loop
+                        age = None
+                    
+                    if age is not None and age > STALE_LOCK_SECONDS:
+                        try:
+                            logger.warning(
+                                "Stale lock detected for %s (age %.0fs). Reclaiming lock...",
+                                path, age
+                            )
+                            lock_path.unlink()
+                        except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)